README.md

```diff
@@ -1 +1,30 @@
-# LeaderBoard-Api
\ No newline at end of file
+# LeaderBoard-Api
+## Project Name
+"LeaderBoard-Api"  is a project design to send and receive data to/from the API, its a solo project set up by Microverse as a core requirement the code was broken into smaller fragments called module. 
+this is the basic structure for project
+
+## Built With
+The major languages used for this project are : 
+HTML,
+Javascript 
+
+## Technologies used
+The amazing tecnologies for this project are:
+Linters,
+git
+gitflow
+webpack
+ ## How to  make a copy of this project(clone):
+ Visit our github url listed below
+ search project(repo name)
+ use git fork and pull to ur locall machine or download the zip code .. 
+## Author:
+
+- ## ISIGBA AMBROSE
+ ## Social:
+ Github:[@ambrosegithub](https://github.com/Ambrosegithub)
+-Twitter:[@Glorious12](https:mobile.twitter.com/Glorious851)
+ Facebook:Isigba Ambrose Davaid
+ Email: isigba.ammbrose94@gmail.com
+ Whatsapp tel: +393512463798
+ Contact: +3938817911630, +393298582707.
```

Comment:
- [OPTIONAL] As requested by the previous code reviewer, please it will be nice to restructure you readme file following this [Microverse template](https://github.com/microverseinc/readme-template), as the will make your readme file more informative and professional. Your readme is the first thing a  user sees when he/she visits your repository, so it gives him/her an impression of the kind of person you are, you never know who is looking, may be a potential recruiter.